manifests/d/Drovio/Drovio/3.2.3/Drovio.Drovio.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Drovio.Drovio
+PackageVersion: 3.2.3
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://repository.drovio.com/stable/drovio/win64/latest_version/drovio-win64.exe

```

Comment:
```suggestion
  InstallerUrl: https://repository.drovio.com/stable/drovio/win64/3.x/3.2.3/drovio-win64.exe
```

Reply:
Hi, latest_version is more relevant since this link will always point to the latest release thus allowing us to only have to update the release number